app/client/rest/channels.ts

```diff
@@ -306,6 +311,25 @@ const ClientChannels = (superclass: any) => class extends superclass {
             {method: 'post', body: JSON.stringify({term})},
         );
     };
+
+    // Channel Category Routes
+    getChannelCategoriesRoute(userId: string, teamId: string) {
+        return `${this.getBaseRoute()}/users/${userId}/teams/${teamId}/channels/categories`;
```

Comment:
Would we ever fetch the categories from another user than the current user? could we just set `userId` as `me` ?

Reply:
I think so - though I'm just following the existing convention in the mobile/web app where userId is passed along.